zebrad/src/config.rs

```diff
@@ -97,6 +92,16 @@ impl TracingSection {
             flamegraph: None,
         }
     }
+
+    pub fn env_filter(&self) -> EnvFilter {
+        self.filter.as_deref().unwrap_or("info").into()
```

Comment:
Issue: I think the default has to be warn here, to avoid conflicts with commands which use standard output.

(For commands that use a config, we change the default in after_config.)

```suggestion
        self.filter.as_deref().unwrap_or("warn").into()
```

Reply:
>  to avoid conflicts with commands which use standard output.

are we talking about non `is_server` commands? If so I don't think this is an issue, the env filter doesn't enable logging, so this will only apply to filtering spans in spantraces.

Reply:
Ah, right, so that's what I'm missing here.

Maybe we could improve the function documentation?